### ISSUE TITLE
Please reopen issue 3019

### ISSUE THREAD

**Issue by blongstreth** [angular-ui/bootstrap#6409]
I have commented on the following on old issue which I think should be reopened please: 
https://github.com/angular-ui/bootstrap/issues/3019

**Comment by wesleycho** [comment]
Going to leave the prior issue closed, as that proposal IMO doesn't provide the most extensibility - however, I would be open to exposing `openedWindows` publicly in the $modalStack service so that one can have maximal flexibility without bloating the library much.

**Comment by blongstreth** [comment]
Exposing `openedWindows` is really what I am after so I can inspect them and then can controlled what can be opened, closed, etc.  Will you be able to add this feature soon please?  If so, will you be creating an issue or do we just track the commits?  Please let me know.

**Comment by blongstreth** [comment]
I saw this project is listed as feature complete.  I hope someone steps in to at least address a few critical issues, etc. down the road :-(

**Comment by mendeza** [comment]
Sorry @blongstreth, but you seem to misunderstand the true nature of the project status.

>As per the README, this project is no longer being maintained. Any issues entered will remain uninvestigated and unresolved.

**Comment by blongstreth** [comment]
No, I understood it.  I was just testing the waters :-) Worst case scenario is I can always fork it and fix what I need should I ever need to. Thanks for the quick reply!
